graderutils_format/templates/default.css

```diff
@@ -25,8 +25,21 @@ pre {
 div.warning-messages {
     margin-top: 35px;
 }
-/* Override stylesheet default badge color inside panel-heading */
+
+/* Override stylesheet badge colors inside panel-heading since otherwise they are black */
+
+/* Error */
 div.grading-task .panel-default > .panel-heading .badge {
     color: #fff;
     background-color: #777;
 }
+/* Success */
+div.grading-task .panel-default > .panel-heading .badge.badge-success {
+    color: #fff;
```

Comment:
I verified with the WAVE plugin that these colours pass the colour contrast requirement for accessibility.